### PR TITLE
Prep the 3.6.0 release

### PR DESCRIPTION
**Pull request by felipeelia**
Prepare the 3.6.0 release, following the steps outlined in our [CONTRIBUTING.md](https://github.com/10up/ElasticPress/blob/develop/CONTRIBUTING.md#release-instructions) file.

- [x] If the new version requires a reindex, add its number to the $reindex_versions array in the ElasticPress\Upgrades::check_reindex_needed() method. If it is the case, remember to add that information to the Changelog listings in readme.txt and CHANGELOG.md.
- [x] Branch: Starting from develop, cut a release branch named release/3.6.0 for your changes.
- [x] Version bump: Bump the version number in elasticpress.php, package.json, readme.txt, and any other relevant files if it does not already reflect the version being released. In elasticpress.php update both the plugin "Version:" property and the plugin EP_VERSION constant.
- [x] Changelog: Add/update the changelog in CHANGELOG.md and readme.txt, ensuring to link the [3.6.0] release reference in the footer of CHANGELOG.md (e.g., https://github.com/10up/ElasticPress/compare/3.5.6...3.6.0).
- [x] Props: Update CREDITS.md file with any new contributors, confirm maintainers are accurate.
- [x] Readme updates: Make any other readme changes as necessary. README.md is geared toward GitHub and readme.txt contains WordPress.org-specific content. The two are slightly different.
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in .gitattributes.
- [x] Change the release date in CHANGELOG.md
- [ ] Merge: Merge the release branch/PR into develop, then make a non-fast-forward merge from develop into master (git checkout master && git merge --no-ff develop). master contains the stable development version.
- [ ] Test: While still on the master branch, test for functionality locally.
- [ ] Push: Push your master branch to GitHub (e.g. git push origin master).
- [ ] Release: Create a new release, naming the tag and the release with the new version number, and targeting the master branch. Paste the release changelog from CHANGELOG.md into the body of the release and include a link to the closed issues on the milestone.
- [ ] SVN: Wait for the GitHub Action to finish deploying to the WordPress.org repository. If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.
- [ ] Check WordPress.org: Ensure that the changes are live on https://wordpress.org/plugins/elasticpress/. This may take a few minutes.
- [ ] Close milestone: Edit the milestone with release date (in the Due date (optional) field) and link to GitHub release (in the Description field), then close the milestone.
- [ ] Punt incomplete items: If any open issues or PRs which were milestoned for 3.6.0 do not make it into the release, update their milestone to 3.6.1, 3.7.0, 4.0.0 or Future Release.
- [ ] Create new branches from `master`: `trunk` and `stable`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,22 +8,56 @@ All notable changes to this project will be documented in this file, per [the Ke
...
+* Custom sorting to features on the Features page. Props [@Rahmon](https://github.com/Rahmon) via [#1987](https://github.com/10up/ElasticPress/pull/1987)
+* Terms: add a new `facet` field to hold the entire term object in json format. Props [@moraleida](https://github.com/moraleida) via [#2071](https://github.com/10up/ElasticPress/pull/2071)
+* Elasticsearch connection check to Site Health page. Props [@spacedmonkey](https://github.com/spacedmonkey) and [@Rahmon](https://github.com/Rahmon) via [#2084](https://github.com/10up/ElasticPress/pull/2084)
+* Support to NOT LIKE operator for meta_query. Props [@Thalvik)](https://github.com/Thalvik) and [@Rahmon](https://github.com/Rahmon) via [#2157.](https://github.com/10up/ElasticPress/pull/2157)
+* Support to `category__not_in` and `tag__not_in`. Props [@oscarssanchez](https://github.com/oscarssanchez) and [@felipeelia](https://github.com/felipeelia) via [#2174](https://github.com/10up/ElasticPress/pull/2174)
```
Comment (brandwaffle) [inline CHANGELOG.md:25]: @felipeelia let's change to "Support for" in these instances

Comment (felipeelia) [reply CHANGELOG.md]: Fixed @brandwaffle. Thanks!

**Comment by jeffpaul** [comment]
@brandwaffle @felipeelia I _think_ we can probably get rid of the `/lang` directory and its contents plus the [Domain Path header variable](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#domain-path) now that [translate.wordpress.org](https://translate.wordpress.org/projects/wp-plugins/elasticpress/) handles translations.

**Comment by felipeelia** [comment]
That makes sense for me @jeffpaul . I've pushed a new commit adding that to the ignored folders.

**Pull request merged**